crypto/keyring/keyring.go

```diff
@@ -522,15 +529,15 @@ func (ks keystore) NewMnemonic(uid string, language Language, hdPath, bip39Passp
 		bip39Passphrase = DefaultBIP39Passphrase
 	}
 
-	info, err := ks.NewAccount(uid, mnemonic, bip39Passphrase, hdPath, algo)
+	ke, err := ks.NewAccount(uid, mnemonic, bip39Passphrase, hdPath, algo)
 	if err != nil {
 		return nil, "", err
 	}
 
-	return info, mnemonic, nil
+	return ke, mnemonic, nil
 }
-
-func (ks keystore) NewAccount(name string, mnemonic string, bip39Passphrase string, hdPath string, algo SignatureAlgo) (Info, error) {
+// TODO keep or remove SignatureAlgo???
```

Comment:
I'm not 100% sure about this. maybe let's keep it?

Reply:
cc @robert-zaremba